docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
<p>I feel the details about Cliffordization can go into the "Cliffordize the PUBs" section, so this intro doesn't feel the tool is all about cliffordization. Maybe we can just say something like "Neat Cliffordizes the input circuits for efficient simulation, while retaining its structure and depth. Clifford circuits suffer similar levels of noise and is a good proxy for studying the original circuit of interest.". And the rest about how it's done can go to the later section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>